### ISSUE TITLE
Enable Lighthouse as an action on all PRs

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4336]
### Background
Now that the accessibility epic is wrapping up, we're in a place to enable Lighthouse accessibility testing on our PRs. We have an existing Lighthouse action, but:
- it runs daily, not against PRs, so we didn't realize it had failures for a long time
- it only tests against a few pages in the app, and doesn't appear to be interactive (it can't click buttons or interact with the page right now).

### Action requested
- Enable the Lighthouse changes to run on individual branches, instead of the demo site
- Ensure that the action fails when accessibility violations are introduced (accessibility score less than 100)

### Acceptance Criteria
Lighthouse is running as a status check on PRs.

### Additional context
To see detailed information about a given lighthouse run, look at the lighthouse-results artifact produced (available in the run summary). You can open the HTML files generated for a detailed look at the failures.
There's a guide to enabling Lighthouse as a Github Action [here](https://www.adrianbolonio.com/blog/accessibility-github-actions) that may be useful.

I also spent some time looking into the testing offered by axe-core (the same folks who make the extension we run manually). I don't think it's worth moving forward with it at this time, because we'd need to render and test components individually (rather like Jest). But we aren't tied to Lighthouse; if there's a better testing tool you come across it's worth investigating.

**Comment by georgehager** [comment]
Thinking through a couple options for this

**Option 1:** Run against localhost on branches (like e2e)
https://github.com/treosh/lighthouse-ci-action#lighthouse-ci-action
https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/getting-started.md#getting-started

Wait to turn on for PR pushes until:
- Have existing seed files that facility id's can easily be grabbed from

Then:
- Set up lighthouse job to start the app with the given seed files (may need a chromium installation)
- Run lighthouse against that local app (like e2e tests)

**Option 2:** Run against static html from local build on branch

- https://github.com/treosh/lighthouse-ci-action#mainyml-3
- super easy but not super useful for a dynamic app - it would likely only run against index.html

**Comment by georgehager** [comment]
Adding some documentation here for future reference:
[lighthouse ci aka lhci](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/getting-started.md#overview)
[lhci configuration](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#configuration)
[the plain not-optimized-for-ci tool aka lighthouse](https://github.com/GoogleChrome/lighthouse/#cli-options)
